src/VisualStudio/VisualBasic/Impl/Options/AdvancedOptionPageStrings.vb

```diff
@@ -200,16 +200,10 @@ Namespace Microsoft.VisualStudio.LanguageServices.VisualBasic.Options
         Public ReadOnly Property Option_SeparateImportGroups As String =
             BasicVSResources.Separate_import_directive_groups
 
-        Public ReadOnly Property Option_Suggest_imports_for_types_in_reference_assemblies As String
-            Get
-                Return BasicVSResources.Suggest_imports_for_types_in_reference_assemblies
-            End Get
-        End Property
+        Public ReadOnly Property Option_Suggest_imports_for_types_in_reference_assemblies As String =
```

Comment:
❓ Why are changes to this file appearing in this pull request

Reply:
Reverting.  sorry!